org.osgi.test.cases.rest.client.js/src/org/osgi/test/cases/rest/client/js/junit/RestJSClientTestCase.java

```diff
@@ -413,6 +413,11 @@ public void testBundleHeaderRestClient() throws Exception {
 				+ "  success : function(res) {");
 
 		for (String key : Collections.list(headers.keys())) {
+			if ("Bundle-Developers".equals(key))
```

Comment:
This just ignores the real problem for one header which causes issues today. I will make a PR with a more complete fix.